client/ayon_core/cli_commands.py

```diff
@@ -20,28 +20,12 @@ def launch_tray():
         tray.main()
 
     @staticmethod
-    def add_addons(click_func):
-        """Modules/Addons can add their cli commands dynamically."""
-
-        from ayon_core.lib import Logger
-        from ayon_core.addon import AddonsManager
-
-        manager = AddonsManager()
-        log = Logger.get_logger("CLI-AddModules")
-        for addon in manager.addons:
-            try:
-                addon.cli(click_func)
-
-            except Exception:
-                log.warning(
-                    "Failed to add cli command for module \"{}\"".format(
-                        addon.name
-                    ), exc_info=True
-                )
-        return click_func
-
-    @staticmethod
-    def publish(path: str, targets: list=None, gui:bool=False) -> None:
+    def publish(
+        path: str,
+        targets: list = None,
+        gui: bool = False,
+        addons_manager=None,
```

Comment:
Should this get a type hint? Also, argument is lacking in the docstring

Reply:
Wanted to move the function to different place in following PR.

Reply:
Life is short, added.